### ISSUE TITLE
System/DnsService stuck in "Deleting" state

### ISSUE THREAD

**Issue by profet23** [Azure/service-fabric-issues#982]
Getting this error for all instances on all nodes for the `fabric:/System/DnsService`:

```
Unhealthy event: SourceId='System.RAP', Property='IStatelessServiceInstance.CloseDuration', HealthState='Warning', ConsiderWarningAsError=false. The api IStatelessServiceInstance.Close on node _vf57e7qon_0 is stuck. Start Time (UTC): 2018-04-16 15:32:53.790.
```

Status is stuck on "Deleting"

Not sure how to proceed. Any help would be appreciated.

This is happening in a remote Azure cluster.

**Comment by ninzavivek** [comment]
- Are you still stuck on this issue? Or the issue has been mitigated?
 - Is this an SFRP cluster?
 - What version of SF are using?

**Comment by profet23** [comment]
After restarting the seed nodes the problem went away... But so did the dnsservice. I had to turn it back on in the resource json. After I did that the dnsservice came back online and everything is fine. Not sure what the issue was.

**Comment by oanapl** [comment]
Thank you for the update, @profet23 . We had a bug in the DnsService"close" that led it to be stuck. The bug is fixed in our incoming 6.2 release. 
Regarding the unexpected behavior of having the Dns service disappear, this is a known issue we are following up on. 
I will close this, please re-open if you have more questions/issues.
